CMakeLists.txt

```diff
@@ -171,10 +192,22 @@ if(USE_CUDA)
   set(CMAKE_CUDA_HOST_COMPILER ${CMAKE_CXX_COMPILER})
   message(STATUS "Configured CUDA host compiler: ${CMAKE_CUDA_HOST_COMPILER}")
 
+  if(POLICY CMP0104)
+    cmake_policy(SET CMP0104 NEW)
```

Comment:
Is there a code path I am not seeing where we aren't getting 3.18 as our minimum cmake version?